templates/package.json.template

```diff
@@ -34,9 +34,10 @@
     "dependencies": {
       "arguejs": "^0.2.3",
       "lodash": "^4.15.0",
+      "long": "^3.2.0",
```

Comment:
It's being used in the `longsAsStrings` option. In Protobuf.JS 5, Long was included in protobuf.js. Now, it's a separate module. Protobuf needs the long dependency, then, to produce Long objects when `longsAsStrings` is unset.